.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Thank you for sending a PR !-->
+
+Please perform the following checks and tick mark all the points.
+
+- [ ] The page (if new), does not already exist in the repo.
+
+- [ ] The page (if new), is added to the correct platform folder.
+i.e. If its common to all platforms, it is under `common/` and if its linux specific, its under `linux/`. If the tool works on all unix-y platforms, please add multiple copies of the same page under all folders.
```

Comment:
These `its` need an apostrophe.

 - "If its common to all platforms" -> "If it's common to all platforms"
 - "and if its linux specific, its under" -> "and if it's linux specific, it's under"